feature-flag.c

```diff
@@ -55,3 +55,7 @@ void feature_flag_load(struct feature_flag *feature_flag, unsigned const char ke
         feature_flag->url_encryption_enabled = !strcmp(ff_url_encryption, "1");
     }
 }
+
+void feature_flag_cleanup() {
+    config_unlink("session_ff_url_encryption");
```

Comment:
Consider creating a macro and reusing that macro, otherwise, this will be 3rd time `session_ff_url_encryption` is repeated.

Example: 

`define URL_ENCRYPTION "session_ff_url_encryption"`